### PR TITLE
Add alerting rules for DeadManSnitch

### PR DESCRIPTION
**Pull request by maulikjs**

### CODE REVIEW

File: monitoring/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -430,6 +430,17 @@ data:
...
 
+      - name: DeadManSnitch
+        interval: 1m
+        rules:
+          - alert: DeadManSwitch
+            expr: vector(1)
+            labels:
+              severity: critical
+            annotations:
+              description: RHODS monitoring and alerting pipeline is offline.
```
Comment (anishasthana) [inline monitoring/prometheus/prometheus.yaml:441]: It should be the other way around right? The description of this alert is that the pipeline _is_ online.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: monitoring/prometheus/prometheus.yaml (hunk 1 above)
Comment (maulikjs) [reply monitoring/prometheus/prometheus.yaml:441]: ah right yes

Comment (maulikjs) [reply monitoring/prometheus/prometheus.yaml]: updated

**Review by anishasthana — APPROVED** [review]
Changes look good, but I'd like for us to actually test it out. Do you have a deployment on stage OSD we can look at?
Can you also make a corresponding PR to https://gitlab.cee.redhat.com/croberts/olminstall for the dms secret creation so that we can have an easy way to continue installing RHODS?

**Comment by maulikjs** [comment]
@anishasthana the secret is created as part of the add-on installation workflow. REF: https://gitlab.cee.redhat.com/service/managed-tenants/-/merge_requests/889

**Comment by anishasthana** [comment]
Right, but can you create a dummy secret so we can test in non-OSD clusters internally.

**Comment by maulikjs** [comment]
Ah ok raising a MR then

**Review by crobby — APPROVED** [review]
seems to work now, lgtm

**Review by crobby — APPROVED** [review]
Seems to be working

**Pull request merged**